network/load-balancer/reference-content/configuring-backends.mdx

```diff
@@ -0,0 +1,132 @@
+---
+meta:
+  title: Configuring your Load Balancer's backend
+  description: Learn how to choose different settings and configurations for your Load Balancer's backend depending on your needs
+content:
+  h1: Configuring your Load Balancer's backend
+  paragraph: Learn how to choose different settings and configurations for your Load Balancer's backend depending on your needs
+tags: load-balancer backend protocol tcp http tls-encryption verify-certificate proxy-protocol balancing-method balancing-rule balancing-algorithm round-robin least-connections first-available stick-sessions timeout tunnel-timeout server-timeout connection-timeout backend-protection max-simultaneous queue-timeout retries retry-policy
+categories: 
+  - load-balancer
+  - network
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-05
+---
+
+## Load Balancer Overview
+
+<Macro id="load-balancer" />
+
+## Backend overview
+
+Each Load Balancer is configured with one or several backends. A backend represents a set of backend servers that the Load Balancer’s frontend(s) forwards requests to using the specified configuration (port, protocol, proxy protocol etc).
+
+You can create frontends and backends [at the same time as creating the Load Balancer itself](/network/load-balancer/how-to/create-frontends-backends/#how-to-create-a-backend-during-creation-of-your-load-balancer), or create an "empty" Load Balancer and [create/add frontends and backends later](/network/load-balancer/how-to/create-frontends-backends/#how-to-create-a-backend-after-creation-of-your-load-balancer). 
+
+You have access to the same configuration settings for your backend whether you create it with the Load Balancer, or later. Any of these settings can be [edited](/network/load-balancer/how-to/manage-frontends-and-backends/#how-to-edit-backends) thereafter. Read on to find out more about the different configuration settings available for your backend.
+
+## Configuring basic settings
+
+### Backend name
+
+You can use an automatically-generated random name suggested by the console, or choose your own name for the backend
+
+### Protcol and port
+
+The Load Balancer initiates connections to its backend servers using the protocol and port you define. 
```

Comment:
@RoRoJ, since HTTP is built on top of TCP, in any case the load balancer will initiate TCP connections to the backend server. So I wonder if we could phrase this a bit differently:

"The Load Balancer forwards traffic to its backend servers using the protocol and port you define, and can therefore leverage selected protocol's capabilities to manage traffic"